clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -982,6 +986,22 @@ private static void applyDerivedSchemaToStore(CommandLine cmd) throws Exception
     printObject(valueResponse);
   }
 
+  private static void generateDerivedSchemaForStore(CommandLine cmd) throws Exception {
```

Comment:
This is good, but I am wondering is it better to directly register it to the ZK? Because I believe you want to register to ZK eventually....

Reply:
There is another command already register derived schema to ZK, maybe we could update it instead.

Reply:
So I've held off on doing that because to my knowledge we haven't resolved the gotchas of this derived schema with incremental push deletes?  Same for @FelixGV's comment.  If I'm wrong I'll make the change

Reply:
Ya know what.  I'll meet you half way.  I'll print the schema and have a [Y/N] prompt for the admin if they want to apply the schema or not.

Reply:
Added.